enhancements/ingress/bind-options.md

```diff
@@ -0,0 +1,268 @@
+---
+title: bind-options
+authors:
+  - "@m-yosefpor"
+reviewers:
+  -
+approvers:
+  -
+creation-date: 2021-08-04
+last-updated: 2021-08-04
+status: implementable
+see-also:
+  - https://github.com/openshift/cluster-ingress-operator/issues/633
+  - https://github.com/openshift/api/issues/964
+replaces:
+superseded-by:
+---
+
+# Ingress Bind Options
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement extends the IngressController API to allow the user to
+specify bindOptions in HostNetwork strategy and enables them to run multiple
+instances of ingresses on the same node in the HostNetwork strategy, with
+mitigating port binding conflicts.
+
+## Motivation
+
+When using HostNetwork strategy for ingressControllers, the default 80, 443, 10443, 10444, 1936 ports on the host are binded on HAProxy for http, https, no_sni, sni and stats correspondingly. However, those ports might be occupied by other processes (such as another set of ingressControllers), which makes it impossible to run multiple set of ingressControllers on the same nodes with HostNetwork strategy. In OKD3.11, it was possible to listen on custom host ports via setting these env vars in Routers DeploymentConfig, however there is not any options to specify custom ports in `IngressController.operator.openshift.io/v1` object right now.
```

Comment:
Can you mention which environment variables were used in OpenShift 3.11? Links to the documentation would also we helpful.

Also, some minor remarks: `s/binded on HAProxy/bound by HAProxy/`, `s/to run on multiple set/to run on multiple sets/`, `s/OKD3.11/OpenShift 3.11/`.

Reply:
- env vars mentioned
- link to openshift 3.11 documentation added
- typos fixed